addons/mail/static/src/messaging/test_utils.js

```diff
@@ -60,7 +60,7 @@ const MockMailService = Class.extend({
         return AbstractStorageService.extend({ storage: new RamStorage() });
     },
     messaging() {
-        return MessagingService;
+        return MessagingService.extend({});
```

Comment:
Already fixed by https://github.com/odoo-dev/odoo/pull/230

Reply:
Will remove when #230 is merged, just keeping it to have green runbot

Reply:
The relevant part was merged now.

Reply:
Currently rebasing and removing that parts